src/SharpGLTF.Ext.3DTiles/Schema2/Ext.StructuralMetadataRoot.cs

```diff
@@ -95,9 +95,16 @@ public StructuralMetadataSchema UseEmbeddedSchema(string id)
                 return schema;
             }
 
+            // Sets the schema to use an external schema, returns an empty schema to used for adding schema properties
+            public StructuralMetadataSchema UseExternalSchema(Uri uri)
```

Comment:
here the method for setting external schema uri, it works but doesn't use a ISchemaProxy

Reply:
external schema is not supported for now